vendor/ddcloud/resource_vip_pool_member.go

```diff
@@ -45,10 +45,9 @@ func resourceVIPPoolMember() *schema.Resource {
 				Computed: true,
 			},
 			resourceKeyVIPPoolMemberPort: &schema.Schema{
-				Type:     schema.TypeString,
```

Comment:
Ok, just to confirm - if you omit the `port` attribute entirely from the `ddcloud_vip_pool_member` resource does it create the membership with no port?